packages/esbuild/test/e2e/rebuild/rebuild.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from 'chai';
+import { ESBuildTestKit } from '../esbuild-testkit';
+import { sleep } from 'promise-assist';
+
+describe('Stylable ESBuild plugin rebuild on change', () => {
+    const tk = new ESBuildTestKit();
+
+    afterEach(() => tk.dispose());
+
+    it('should pick up rebuild', async () => {
+        const { context, read, write } = await tk.build({
+            project: 'rebuild',
+            tmp: true,
+        });
+        const css1 = read('dist/index.js');
+        expect(css1, 'initial color').to.includes('color: red');
+        await context.watch();
+        await sleep(2222);
```

Comment:
is there a way to wait for build to be done without sleeping for 4.5s?

Reply:
not that I know. he state the the watch loop is at most 2s. so I put little more